# Suppresses flags not supported by the version of clang we are using

--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1533,9 +1533,6 @@ config("default_warnings") {
 
           # TODO(https://crbug.com/1050281): Clean up, enable.
           "-Wno-non-c-typedef-for-linkage",
-
-          # TODO(https://crbug.com/1114873): Clean up, enable.
-          "-Wno-string-concatenation",
         ]
 
         cflags_c += [
